environment.yml

```diff
@@ -1,8 +1,19 @@
+name: extra_foam
+
 channels:
   - conda-forge
   - anaconda
   - defaults
+
 dependencies:
-  - cmake>=3.18
+  - gxx_linux-64>=7
+  - libstdcxx-ng
+  - cmake>=3.17
   - numpy>=1.19
-  - tbb-devel<2021.1
+  - tbb-devel>=2021
+  - tbb>=2021
+  - xtensor
+  - xtensor-python
+  - xtensor-blas
+  - xsimd
+  - pybind11
```

Comment:
Now that the GitHub Workflows are working it's worth creating a lock file with pinned dependencies and setting up Dependabot to automatically update them, this way we can automatically detect issues caused by dependency upgrades before they come up in some random PR or issue from a user.